### ISSUE TITLE
How to import css correctly?

### ISSUE THREAD

**Issue by geekplux** [helion3/inspire-tree#141]
I import the css using following code:

```js
import 'inspire-tree/dist/inspire-tree.css';
```

but some errors occured because it can't find the images.
So how to import css correctly?

**Comment by viveleroi** [comment]
Not sure what's causing problems, we use only one image and the CSS file refers to the file using a relative url. You can always try importing the source SCSS, especially if you already compile scss in your project.

**Comment by geekplux** [comment]
Did you bundle the css and images into bundle.js by webpack?

**Comment by geekplux** [comment]
Do I need to import the css? but if I don't import the css, there is no style..

**Comment by viveleroi** [comment]
I'm not sure that I've tested importing the compiled css into another webpack project but I don't see any reason it wouldn't work. It likely depends on how you've setup webpack. Our CSS is not distributed inline (with any js), it's a separate css file with only one external GIF.

I have bundled the source SCSS in other webpack projects though, so I can say for sure that works, just needs something to compile it.
